lib/improver/nbhood/nbhood.py

```diff
@@ -198,7 +200,7 @@ def check_cube_coordinates(cube, new_cube):
         return new_cube
 
 
-class NeighbourhoodProcessing(object):
+class BaseNeighbourhoodProcessing(object):
     """
     Apply a neighbourhood processing method to a thresholded cube.
```

Comment:
I think this could have a slightly more detailed docstring, to explain it can't be called on its own or something?

Reply:
Done.